code/game/objects/items/AI_modules.dm

```diff
@@ -357,8 +365,36 @@ AI MODULES
 		law_datum.clear_zeroth_law(0)
 	..()
 
+/obj/item/ai_module/core/full/handle_unique_ai()
+	var/datum/ai_laws/unique_ai_laws =  GLOB.round_default_lawset
+	if(law_id == initial(unique_ai_laws.id))
+		return
+	return QDEL_MODULE
+
 /******************** Asimov ********************/
 
+/obj/item/ai_module/core/round_default_laws
+	name = "Default Laws" //gets renamed
```

Comment:
maybe make this a spawner instead so that it gets deleted either way, that way it isn't confusing.

Reply:
It needs to have something exist to be the round default if it needs to fallback, aka a /datum/lawset that doesn't have a module object with a matching id.

Reply:
you can then spawn the fallback instead

Reply:
Alright if you're sure